packages/react/src/components/heliopay-container/index.tsx

```diff
@@ -166,12 +173,20 @@ const HelioPayContainer: FC<HeliopayContainerProps> = ({
         onError: handleErrorPayment,
         onPending,
         customerDetails,
+        quantity: Number(quantity),
         productDetails,
-        quantity: Number(quantity) ?? 1,
+        splitRevenue: paymentDetails?.features?.splitRevenue,
+        splitWallets: paymentDetails?.splitWallets,
+        wallet: wallet as AnchorWallet,
```

Comment:
doesn't it come with AchorWallet type ?

Reply:
yes, but we have wallet: AnchorWallet? here. let's fix this during refactoring